bluepyemodel/model/model_configurator.py

```diff
@@ -52,7 +52,9 @@ def new_configuration(self, use_gene_data=False):
             self.configuration = NeuronModelConfiguration(
                 distributions=self.access_point.get_distributions(),
                 available_mechanisms=self.access_point.get_available_mechanisms(),
-                available_morphologies=self.access_point.get_available_morphologies(),
+                available_morphologies=self.access_point.get_available_morphologies(
+                    skip_get_available_morph=False
```

Comment:
why do you have 'skip_get_available_morph' here?

Reply:
Because this is used in bluepyemodel.model.model_configuration.configure_model that later needs available_morphologies in select_morphology, and because I added the possibility of skipping the availables morphologies for the nexus access point in a PR on bpemnexus

Reply:
ok but `get_available_morphologies` doesn't seem to have any `skip_get_available_morph` parameter in its definition

Reply:
Oh yes, thank you. removed in latest commit